src/index.ts

```diff
@@ -20,7 +20,15 @@ export default class MSSQLConnector extends BaseConnector {
     this.pool = new this.client.ConnectionPool(this.configOptions)
   }
 
+  onStop(): void {
+    this.close()
+  }
+
   // Your actions
+  getConenctionPool() {
```

Comment:
Should remove the `// Your actions` line - as this is just from the template I think